crates/ursa-telemetry/src/lib.rs

```diff
@@ -11,7 +11,7 @@ pub struct TelemetryConfig {
     /// Service name.
     pub name: String,
     /// Service log level.
-    pub log_level: Option<String>,
+    pub log_level: String,
```

Comment:
should stay optional, in case it is not provided it will default to info

Reply:
sounds good

Reply:
https://github.com/fleek-network/ursa/pull/223/commits/acc01e7fc57f79afcd4300e3172bb6b462b7fc1a